docs/reference/README.md

```diff
@@ -0,0 +1,21 @@
+# API Reference
+
+This is the API Reference for Piraeus Operator. It lists user settable options that control the state of
+Piraeus Datastore.
+
+### [`LinstorCluster`](./linstorcluster.md)
+
+This resource controls the state of the LINSTOR&reg cluster and integration with Kubernetes.
+
+### `LinstorSatelliteConfiguration`
+
+*Coming soon*
+
+This resource controls the state of the LINSTOR Satellites, optionally applying it to only a subset of nodes.
+
+### `LinstorSatellite`
+
+*Coming soon*
+
+This resource controls the state of a single LINSTOR Satellite. This resource is not intended to be changed directly,
```

Comment:
If this reference lists "user settable options" then we don't need to document this CR. Maybe we should still document whatever useful information can be read from there. In that case, we should change the paragraph at the top.

Reply:
Good point. Reworded the top a bit. Also: added the status fields on the LinstorCluster to set a precedent.